_posts/2024-09-25-cockpit-325.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 325
+author: martinpitt
+date: '2024-09-25'
+tags: cockpit, machines
+slug: cockpit-325
+category: release
+summary: 'VM snapshot improvements, VM descriptions'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 325 and cockpit-machines 320:
+
+## Machines: Improvements to snapshots of running machines
+
+The snapshot of a running machine includes its memory state, and Cockpit sometimes asks where to store it. This is now done in a slightly improved fashion: you only need to specify the directory, and Cockpit will provide the name of the file. The default for this directory now also takes previous snapshots into account, so that you don't need to touch it if you always store the memory state of all snapshots in the same directory. Plus, Cockpit now gives estimates for the amount of space that will be needed and warns you if there doesn't seem to be enough.
```

Comment:
There are a lot of features; what if we made it a list?

```suggestion
## Machines: Snapshot improvements

Snapshotting running machines has improved in several ways:

- Snapshots now include memory state
- Only a directory needs to be specified; Cockpit will generate a filename
- The snapshot dialog remembers the storage path, making future snapshots easier
- Space estimates are shown, including a warning if there isn't enough storage available
```

Reply:
> Snapshots now include memory state

That is not a new feature, that is just a description of the existing feature. Hence, I changed it to a list, but with adjusted wording.